flashcards/backend/decks/tests/test_models_difficulty_level.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from django.db import IntegrityError
+
+from decks.models import DifficultyLevel
+
+
+def test_difficulty_level_gui_representation():
+    difficulty_level = DifficultyLevel(name="hard", value=1)
+    assert str(difficulty_level) == difficulty_level.name
+
+
+def test_difficulty_level_in_db(difficulty_level):
```

Comment:
You are not overriding save method on model, so testing creating database entry is a bit pointless now😅, or are you overriding it?

Reply:
I think Pawel wanted me to change it back  to overwriting save method something that you suggested in the past :
```   
 def save(self, *args, **kwargs):
        name = DifficultyLevel.objects.filter(name__iexact=self.name)
        if not name:
            super(self).save(*args, **kwargs)
        return name
```
I have even prepared new commit for that :)

Reply:
well then pus it :D

Reply:
Its on now, let me now if you like it more?